components/Tabs/style.scss

```diff
@@ -0,0 +1,28 @@
+.TabGroup {
+  .nav {
```

Comment:
Small detail: these child element classes should use BEM-style selectors, so `.TabGroup-nav` and `.TabGroup-tab` [per our SCSS selector conventions](https://next.readme.ninja/ui/#/Conventions/Writing%20CSS "Scroll to the bottom of this page.")!